### ISSUE TITLE
Remove array alignment rule for PHP?

### ISSUE THREAD

**Issue by rmccue** [humanmade/coding-standards#47]
In #41 we removed the object property alignment rule for JS, making this valid code:

```js
const x = {
	foo: "bar",
	longerPropertyName: "value",
};
```

Should we do the same for PHP? Currently, this code fails:
```php
$x = [
	'foo' => 'bar',
	'longerPropertyName' => 'value',
];
```

The error is:
```
Array double arrow not aligned correctly; expected 16 space(s) between "'foo'" and double arrow, but found 1.
(WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
```

**Comment by peterwilsoncc** [comment]
> Should we do the same for PHP?

I'm a :+1:, to reduce diff sizes and misleading blame annotations.

**Comment by rmccue** [comment]
I think that's a good point: constant realignment causes diffs to be all sorts of messed up, which is one of the key reasons we use trailing commas. Makes sense to be consistent with that IMO.

**Comment by rmccue** [comment]
This actually appears to be a regression, as we used to allow it (based on what I'm seeing in H2), so committing and shipping.

**Comment by rmccue** [comment]
Reopening: should we also remove the assignment alignment, which is new in 0.4? (Thanks WPCS for changing these. 🤢 )

**Comment by mikeselander** [comment]
This was fixed in #49, and subsequently #96
